src/sentry/preprod/api/endpoints/project_preprod_list_builds.py

```diff
@@ -46,6 +48,8 @@ def get(self, request: Request, project) -> Response:
         :qparam string build_version: filter by build version
         :qparam string build_configuration: filter by build configuration name
         :qparam string platform: filter by platform (ios, android, macos)
+        :qparam string release_version: filter by release version (formats: "app_id@version+build_number" or "app_id@version")
```

Comment:
Might be good to add some tests for this endpoint now that it'll be used in prod. Can also help to validate the versions are being parsed properly